media/windows/ricoh-printer/ricoh_db_create.py

```diff
@@ -0,0 +1,122 @@
+#!/user/bin/env python3
+
+# Script which runs daily to update staff print totals
+# based on CSVs generated by the printer itself.
+
+import sys
+sys.path.insert(1, '../../python/')
+
+import csv
+import datetime
+import sqlite3
+from sqlite3 import Error
+
+#import ECC
+#import PDSChurch
```

Comment:
Might as well delete the `PDSChurch` import, but the `ECC` one could be useful -- there's handy logging setup stuff in `ECC.py`.